doc/tools/build.py

```diff
@@ -118,7 +118,7 @@ def find_godot(godot, bindir):
     else:
         filemask = 'godot.*.[36][24]'
 
-    binfiles = glob.glob(bindir+filemask)
+    binfiles = glob.glob(str(bindir/filemask))
```

Comment:
`os.path.join(bindir, filemask)`?

Reply:
`The slash operator helps create child paths, similarly to os.path.join():`
https://docs.python.org/3/library/pathlib.html#operators

Python3 complained about bindir+filemask and wouldn't concatenate them. I read a recommendation to use /, but then it complained about variables not being a string. Even though yeah they're supposed to be strings, it said one was a path object.

`binfiles = glob.glob(os.path.join(bindir,filemask))` also works and is now being used.